subscription-engine/kafka-ws/index.js

```diff
@@ -48,13 +50,17 @@ if (config.server.tls) {
   server = http.createServer(application.expressApp);
 }
 
+// register service in stateManager
+StateManager.registerService(stateService);
```

Comment:
Register the services in the `StateManager constructor` because this will never change.

Reply:
Great!